Dockerfile.template

```diff
@@ -0,0 +1,13 @@
+#base-image for node on any machine using a template var
+FROM resin/%%RESIN_MACHINE_NAME%%-node
+
+# install deps on build server, making sure to clean up the artifacts it creates in order to reduce the image size.
+RUN npm install express && npm cache clean && rm -rf /tmp/*
+
+# copy all files to /usr/src/app  dir
+COPY . /usr/src/app 
+
+# Run server when container runs on device
```

Comment:
WORKDIR can be useful, but using absolute paths will work whatever the WORKDIR is set to and means that if they copy/paste just these bits it should continue working or be obvious why it doesn't, as opposed to relying on WORKDIR (although for advanced cases it can make life easier if you know what you're doing - but as a first intro it has confused people before)